<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>649</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>videoProcesserGUI</string>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>灰度值</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
            <item>
             <widget class="QLabel" name="VideoLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
              <item>
               <widget class="QPushButton" name="playBtn">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../icon/play.png</normaloff>../icon/play.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="videoSlider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="ratioLabel">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                 </font>
                </property>
                <property name="text">
                 <string>Frame 0 of 0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0,2">
            <item>
             <widget class="QGroupBox" name="parameters_groupBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="font">
               <font>
                <family>Microsoft YaHei UI</family>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="title">
               <string>参数</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="3">
                  <widget class="QLineEdit" name="LineEdit_y1">
                   <property name="focusPolicy">
                    <enum>Qt::ClickFocus</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Y1</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>X1</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="3">
                  <widget class="QLineEdit" name="LineEdit_y2">
                   <property name="focusPolicy">
                    <enum>Qt::ClickFocus</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Y2</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="LineEdit_x1">
                   <property name="focusPolicy">
                    <enum>Qt::ClickFocus</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="LineEdit_x2">
                   <property name="focusPolicy">
                    <enum>Qt::ClickFocus</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>X2</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="selectRoiBtn">
                 <property name="text">
                  <string>从当前帧选择ROI</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="font">
               <font>
                <family>Microsoft YaHei UI</family>
               </font>
              </property>
              <property name="title">
               <string>画笔</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="1">
                  <widget class="QComboBox" name="comboBox_Color"/>
                 </item>
                 <item row="1" column="2">
                  <widget class="QLabel" name="label_brush_width">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                   <property name="textFormat">
                    <enum>Qt::MarkdownText</enum>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QLabel" name="label_brush_color">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>画笔宽度</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>画笔颜色</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QSlider" name="BrushWidthSlider">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="font">
               <font>
                <family>Microsoft YaHei UI</family>
               </font>
              </property>
              <property name="title">
               <string>视频</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QCheckBox" name="ingray_checkBox">
                 <property name="text">
                  <string>以灰度图播放视频</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="rply_checkbox">
                 <property name="text">
                  <string>自动重播</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="behaviors_groupBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="font">
               <font>
                <family>Microsoft YaHei UI</family>
               </font>
              </property>
              <property name="title">
               <string>功能</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QCheckBox" name="checkBox_offset">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>分析时抵消平均亮度影响</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="analysisBtn">
                 <property name="text">
                  <string>分析</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>...</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openAct"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openAct"/>
   <addaction name="separator"/>
  </widget>
  <action name="openAct">
   <property name="icon">
    <iconset>
     <normaloff>../icon/files.png</normaloff>../icon/files.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="settingAct">
   <property name="icon">
    <iconset>
     <normaloff>../icon/setting.png</normaloff>../icon/setting.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
